### DISCUSSION TITLE
Unable to remove Postgresql beta

### DISCUSSION THREAD

**Discussion by jsbroks** [vercel/community#4378] [Help] [answered]
### Summary

I am unable to access or delete my postgres storage. 

### Steps to Reproduce

<img width="1270" alt="image" src="https://github.com/vercel/community/assets/24920936/468f3ce9-2b4d-44ca-83d4-d1dd4c95b7eb">

Clicking on it results in 404:

<img width="1270" alt="image" src="https://github.com/vercel/community/assets/24920936/0eb6b0c7-29d5-4899-8884-bbf8d20e6a6a">

Yes I have the right permissiosn

<img width="1006" alt="image" src="https://github.com/vercel/community/assets/24920936/0cfdfca1-1360-48ce-b1ff-27dce8282f3a">

**Comment by correttojs** [comment]
hey @jsbroks , apologies for the late reply. We are actively investigating this issue, and we will reach out to you as soon as we have a solution ready for you

**Reply by correttojs** [reply, marked as answer]
hi @jsbroks , happy to communicate that we restored your database. You should now be able to access it from the Vercel dashboard. 
Note that the connection string has been updated.

**Comment by mluttmann** [comment]
Hi @correttojs , I have the same issue described above after deleting my postgres database. Clicking on the broken database redirects me to [this site](https://vercel.com/dashboard/stores/postgres/store_IJf3dAoMV7i3d6QZ/data) with 404. What other information do you need to restore/delete the database?

**Reply by amyegan** [reply]
Thanks for reaching out. It should be cleaned up now. Please let me know if you still have trouble with that database
